.github/scripts/bash/branch_checks.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+check_branch_pr_format() {
+  if echo "${HEAD_BRANCH}" | grep -Eq '^revert-'; then
+    echo "Revert branch, skipping Format check"
+    exit 0
+  fi
+  if echo "${HEAD_BRANCH}" | grep -Eq "^(${SUPPORTED_JIRA_PROJECTS})"; then
+    echo "Branch name starts with 'TS-' or 'SRE-' or 'TDEVOPS-' or 'TOPS-'"
+  else
+    echo "Branch name does not start with ${SUPPORTED_JIRA_PROJECTS}. Checking PR title format."
+    PULL_REQUEST_TITLE="${PR_TITLE}"
+    PATTERN="(${SUPPORTED_JIRA_PROJECTS})-[0-9]+\s\|\s"
+    if [[ ! $PULL_REQUEST_TITLE =~ $PATTERN ]]; then
+      echo "Error: Pull request title is not in the required format. Please use ${SUPPORTED_JIRA_PROJECTS}-XXXX format."
+      exit 1
+    else 
+      echo "PR Title is in the valid format."
+    fi  
+  fi
+}
+
+check_double_commit() {
+  if ([[ "${USER}" != "cipipelinetessell" ]]) && ([[ "${HEAD_BRANCH}" == *"double_commit"* ]] || [[ "${PR_TITLE}" == *"Double Commit"* ]]); then
```

Comment:
Let us use the variable/secret instead of hardcoded `cipipelinetessell` ; And this change to be done wherever there is hardcoded value.

Currently I see `CIPIPELINE_GITHUB_USER` and `CIPIPELINE_USER_NAME` as two users with `CIPIPELINE_GITHUB_TOKEN` and `CIPIPELINE_USER_PASSWORD` holding their respective passwords.

Not sure what is difference between them (if any). Please check and use appropriate variable. Also if any one set can be removed lets do that cleanup as well.